prereise/gather/winddata/hrrr/grib.py

```diff
@@ -0,0 +1,117 @@
+class GribRecordInformation:
```

Comment:
Personal preference, but consider shortening this to `GribRecordInfo`

Reply:
done